src/main/java/com/laserfiche/repository/api/clients/EntriesClient.java

```diff
@@ -42,8 +46,7 @@ CompletableFuture<ODataValueContextOfIListOfFieldValue> getFieldValues(String re
      * @param maxPageSize Optionally specify the maximum number of items to retrieve.
      * @return CompletableFuture<ODataValueContextOfIListOfFieldValue> The return value
      */
-    CompletableFuture<ODataValueContextOfIListOfFieldValue> getFieldValuesNextLink(String nextLink,
-            Integer maxPageSize);
```

Comment:
Would you please run "Reformat Code" on the generated files before commit, so that these line break changes do not appear in the diff, making the review easier.

Reply:
@sxu1998 Remember we have a XML file that represents the style.

Reply:
I'll send the style xml to you in Teams. This needs to be imported to Intellij for "Reformat code"